DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoReceiveBinding.cs

```diff
@@ -51,7 +52,7 @@ public async Task Receive(string modelCardId)
       }
 
       // Receive host objects
-      IEnumerable<string> receivedObjectIds = await unitOfWork.Service
+      IReadOnlyList<ConversionResult> conversionResults = await unitOfWork.Service
```

Comment:
Probably, we will have a different structure on send and receive since we also return `ObjectReference`s on send, so maybe it would be nice to name class explicitly as `ReceiveConversionResult`